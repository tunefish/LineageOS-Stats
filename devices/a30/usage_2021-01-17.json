{
  "17.1": 27,
  "18.0": 587,
  "18.1": 103,
  "none": 4
}