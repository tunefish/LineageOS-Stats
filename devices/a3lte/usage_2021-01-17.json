{
  "15.1": 452,
  "16.0": 141,
  "17.1": 504
}