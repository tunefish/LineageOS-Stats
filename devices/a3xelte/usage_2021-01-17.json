{
  "15.1": 92,
  "16.0": 385,
  "17.1": 2687,
  "18.0": 3,
  "18.1": 7,
  "52.8": 1,
  "none": 2
}