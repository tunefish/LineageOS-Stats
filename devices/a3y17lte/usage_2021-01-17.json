{
  "14.1": 5345,
  "15.1": 265,
  "16.0": 810,
  "17.1": 719,
  "18.0": 1
}