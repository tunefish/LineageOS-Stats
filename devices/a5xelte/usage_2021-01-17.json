{
  "15.1": 35,
  "16.0": 455,
  "17.1": 3303,
  "18.0": 12,
  "18.1": 1,
  "none": 1
}