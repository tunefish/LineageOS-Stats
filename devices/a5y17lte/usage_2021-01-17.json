{
  "14.1": 182,
  "15.1": 115,
  "16.0": 638,
  "17.1": 3641,
  "none": 1
}