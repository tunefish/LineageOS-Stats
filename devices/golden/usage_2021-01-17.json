{
  "13.0": 863,
  "14.1": 2659,
  "15.1": 4,
  "17.1": 2,
  "none": 1
}